src/index.html

```diff
@@ -4,9 +4,283 @@
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="header__logo">
+          <img
+            src="images/logo.png"
+            alt="MOYO-logo"
+            class="header__img"
+          >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="https://www.apple.com/fr/"
+                class="nav__link is-active"
+                target="_blank"
+              >
+              apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+              laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+
+        <div class="catalog__card card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__description">Product code: 195434</p>
+          <div class="card__reviews">
+            <ul class="card__stars stars stars--4">
+              <li class="stars__star star"></li>
+              <li class="stars__star star"></li>
+              <li class="stars__star star"></li>
+              <li class="stars__star star"></li>
+              <li class="stars__star star"></li>
+            </ul>
+            <span>Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span>Price:</span>
+            <span class="card__sum">$2,199</span>
+          </div>
+          <a
+            href="#"
+            data-qa="card-hover"
+            class="card__buy"
+          >Buy
+          </a>
```

Comment:
```suggestion
          <a
            href="#"
            data-qa="card-hover"
            class="card__buy"
          >
            Buy
          </a>
```
Pay attention to formatting. Fix in all places.